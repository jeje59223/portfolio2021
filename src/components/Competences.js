import React from "react";
import "../styles/competences.css"
import html from "../images/html5.png";
import css from "../images/css3.png";
import javascript from "../images/js.png";
import next from "../images/nextjs.png";
import react from "../images/react.png";
import node from "../images/node.png";

const Competences = () => {
    return (
        <section id="competences">
            <div className="container">
                <h2 className="text-center h2Loisirs">Mes Compétences</h2>
                <div className="row align-items-center">
                    <div className="col-md-6 competences">
                        <h2>Comment je me suis formé ?</h2>
                        <hr/>
                        <p>
                            Je suis d'abord et avant tout un passionné ! J'adore apprendre et avant t'intégrer M2I Formations j'ai suivi
                            beaucoup de tutoriel sur internet et me suis formé en autodidacte sur OpenclassNameRoom, Udemy, CodeCademy, H2PROG, etc...
                            J'ai ensuite suivi une formation de Développeur Concepteur d'application et obtenu le certificat professionnel RNCP, niveau II (licence).
                            Enfin j'ai suivi un bootcamp de trois mois chez Fewlines à Lille, qui m'a permis d'intégrer le siège Leroy Merlin en tant sue développeur web.
                        </p>
                    </div>
                    <div className="col-md-6 pt-1">
                        <div className="col text-center">
                            <img src={html} alt="html5" title="html5"/>
                            <img src={css} alt="css3" title="css3"/>
                            <img src={javascript} alt="javascript" title="JavaScript"/>
                        </div>
                        <div className="col text-center">
                            <img src={next} alt="nextJs" title="NextJS"/>
                            <img src={react} alt="react" title="React"/>
                            <img src={node} alt="nodeJS" title="NodeJS"/>
                        </div>
                    </div>
                </div>
            </div>
        </section>
    )
}

export default Competences;