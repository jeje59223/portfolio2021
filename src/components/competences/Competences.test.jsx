import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from "react";
import Competences from "./Competences";

configure({ adapter: new Adapter() });

describe('<Competences>', () => {
  it('should render Competences', () => {
    const wrapper = shallow(<Competences />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});