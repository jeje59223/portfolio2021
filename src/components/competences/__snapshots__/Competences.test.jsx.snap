// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Competences> should render Competences 1`] = `
<section
  id="competences"
>
  <div
    className="container"
  >
    <h2
      className="text-center h2Loisirs"
    >
      Mes Compétences
    </h2>
    <div
      className="row align-items-center"
    >
      <div
        className="col-md-6 competences"
      >
        <h2>
          Comment je me suis formé ?
        </h2>
        <hr />
        <p>
          Je suis d'abord et avant tout un passionné ! J'adore apprendre et avant t'intégrer M2I Formations j'ai suivi beaucoup de tutoriel sur internet et me suis formé en autodidacte sur OpenclassNameRoom, Udemy, CodeCademy, H2PROG, etc... J'ai ensuite suivi une formation de Développeur Concepteur d'application et obtenu le certificat professionnel RNCP, niveau II (licence). Enfin j'ai suivi un bootcamp de trois mois chez Fewlines à Lille, qui m'a permis d'intégrer le siège Leroy Merlin en tant sue développeur web.
        </p>
      </div>
      <div
        className="col-md-6 pt-1"
      >
        <div
          className="col text-center"
        >
          <img
            alt="html5"
            src="html5.png"
            title="html5"
          />
          <img
            alt="css3"
            src="css3.png"
            title="css3"
          />
          <img
            alt="javascript"
            src="js.png"
            title="JavaScript"
          />
        </div>
        <div
          className="col text-center"
        >
          <img
            alt="nextJs"
            src="nextjs.png"
            title="NextJS"
          />
          <img
            alt="react"
            src="react.png"
            title="React"
          />
          <img
            alt="nodeJS"
            src="node.png"
            title="NodeJS"
          />
        </div>
      </div>
    </div>
  </div>
</section>
`;
